rpcs3/Emu/RSX/Program/ProgramStateCache.cpp

```diff
@@ -751,7 +758,7 @@ bool fragment_program_compare::operator()(const RSXFragmentProgram& binary1, con
 	const void* instBuffer1 = binary1.get_data();
 	const void* instBuffer2 = binary2.get_data();
 	usz instIndex = 0;
-	while (true)
+	while (instIndex < (binary1.ucode_length / 16))
```

Comment:
You can easily do a normal for loop here.
Also not sure if it does anything, but you could also put the max index in a local variable.
```suggestion
	for (usz instIndex = 0; instIndex < (binary1.ucode_length / 16); instIndex++)
```